src/gui/ObservingListCommon.hpp

```diff
@@ -21,42 +21,44 @@
 #define OBSERVINGLISTCOMMON_H
 
 #include "Nebula.hpp"
-#include "StelTranslator.hpp"
 #include <QStringList>
 
-struct observingListItem {
-    QString name;
-    QString nameI18n;
-    QString type;
-    QString objtype;
-    QString ra;
-    QString dec;
-    QString magnitude;
-    QString constellation;
-    double jd;
-    QString location;
-    double fov;
-    bool isVisibleMarker;
+struct observingListItem
+{
+	QString name;
+	QString nameI18n;
+	QString type;
+	QString objtype;
+	QString ra;
+	QString dec;
+	QString magnitude;
+	QString constellation;
+	double jd;
+	QString location; // a full location encoded with StelLocation::serializeToLine()
+	QString landscapeID; // landscapeID of landscape at moment of item creation.
+	double fov;
+	bool isVisibleMarker;
 };
 
-Q_DECLARE_METATYPE (observingListItem)
+Q_DECLARE_METATYPE(observingListItem)
 
 enum ObsListColumns {
-    ColumnUUID,        //! UUID of object
-    ColumnName,        //! Name or designation of object
-    ColumnNameI18n,        //! Localized name of object
-    ColumnType,     //! Type of the object
-    ColumnRa,      //! Right ascension of the object
-    ColumnDec,     //! Declination of the object
-    ColumnMagnitude,    //! Magnitude of the object
-    ColumnConstellation,    //! Constellation in which the object is located
-    ColumnDate,   //! Date
-    ColumnLocation,     //! Location where the object is observed
-    ColumnCount        //! Total number of columns
+	ColumnUUID,          //! UUID of object
+	ColumnName,          //! Name or designation of object
+	ColumnNameI18n,      //! Localized name of object
+	ColumnType,          //! Type of the object
+	ColumnRa,            //! Right ascension of the object
+	ColumnDec,           //! Declination of the object
+	ColumnMagnitude,     //! Magnitude of the object
+	ColumnConstellation, //! Constellation in which the object is located
+	ColumnDate,          //! Date
+	ColumnLocation,      //! Location where the object is observed
+	ColumnLandscapeID,   //! LandscapeID linked to object observation
+	ColumnCount          //! Total number of columns
 };
 
 static constexpr char const *JSON_FILE_NAME = "observingList.json";
-static constexpr char const *FILE_VERSION = "2.0";
+static constexpr char const *FILE_VERSION = "2.1";
```

Comment:
I'm not sure it's necessary to increment the version. This is the second version of the observation lists.

Reply:
OK, reverted.